transformer/task.py

```diff
@@ -33,6 +34,7 @@ def from_request(cls, r: Request) -> "LocustRequest":
         return LocustRequest(
             method=r.method,
             url=repr(r.url.geturl()),
+            name=repr(r.name if r.name else r.url.geturl()),
```

Comment:
With the move from `str` to `Optional[str]` I discussed above, I think we can keep `None` if `r.name` is None and let `as_locust_action` deal with `self.url` being the default value.

```suggestion
            name=r.name,
```